supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesInitializationActor.scala

```diff
@@ -81,18 +84,18 @@ class JesInitializationActor(jesParams: JesInitializationActorParams)
   } yield JesBackendInitializationData(jesWorkflowPaths, runtimeAttributesBuilder, jesConfiguration, gcsCreds, genomicsFactory)
 
   override def beforeAll(): Future[Option[BackendInitializationData]] = {
-    def fileUpload(paths: JesWorkflowPaths): Future[Unit] = {
-      writeAuthenticationFile(paths, jesConfiguration.jesAttributes.restrictMetadataAccess, jesConfiguration.dockerCredentials) recoverWith {
-        case CromwellFatalException(e: StorageException) if e.getCode == StatusCodes.PreconditionFailed.intValue && jesParams.restarting =>
-          workflowLogger.debug(s"Authentication file already exists but this is a restart, proceeding.")
-          Future.successful(())
-        case CromwellFatalException(e: StorageException) if e.getCode == StatusCodes.PreconditionFailed.intValue =>
-            Future.failed(new IOException(s"Failed to upload authentication file:" +
-              s" there was already a file at the same location and this workflow was not being restarted: ${e.getMessage}"))
-        case failure => Future.failed(new IOException(s"Failed to upload authentication file", failure))
-      }
+    def fileUpload(paths: JesWorkflowPaths) = existsAsync(paths.gcsAuthFilePath) flatMap {
+      // if it is then something is definitely wrong
+      case true if !jesParams.restarting =>
+        workflowLogger.debug(s"Authentication file already exists but this is a restart, proceeding.")
```

Comment:
looks like this is *not* a restart? and this is failing?

Reply:
Yes if it is **not** a restart it means it should be the first time we ever try to upload the auth file so we definitely don't expect it to be there already.

Reply:
so then this message is not right?

Reply:
Oh yep :) good catch :p